docs/404.html

```diff
@@ -58,7 +58,11 @@
     function replaceNavigationWithVersion(version) {
         var location = document.location.href.split('#')[0];
         var locationSlices = location.split('/');
-        var href = [...locationSlices.slice(0, locationSlices.length - 2), version, 'Readme.html'].join('/');
+        var href = [
+            ...locationSlices.slice(0, locationSlices.length - 2),
+            version,
+            'navigation.html',
```

Comment:
I don't understand why you chose `'navigation.html'` here an not `'documentation.html'`. Is it intentional?

Reply:
No, it's a mistake!